Documentation/contributing/development/dev_setup.rst

```diff
@@ -656,15 +654,10 @@ Minor version
 
 #. Submit all your changes into a new PR.
 
-#. Ping the CI team to make changes in Jenkins (adding new pipeline and
-   dedicated test trigger ``/test-X.XX-4.19`` where ``X.XX`` is the new
-   Kubernetes version).
-
-#. Run ``/test-upstream-k8s`` and the new ``/test-X.XX-4.19`` from the PR once
-   Jenkins is up-to-date.
+#. Ensure that the target CI workflows are running and passing after updating
+   the target k8s versions in the GitHub action workflows.
 
 #. Once CI is green and PR has been merged, ping the CI team again so that they:
-   #. Rotate the Jenkins pipelines and triggers due to removed/added K8s versions.
 
    #. Update the `Cilium CI matrix`_, ``.github/maintainers-little-helper.yaml``,
```

Comment:
There is only one item left in this nested list, so it shouldn't be a list anymore I think?

Reply:
Fixed thanks :+1: